backend/app/helpers.py

```diff
@@ -35,3 +42,20 @@ def __call__(self):
 
 
 aiod_client_wrapper = AIoDClientWrapper()
+
+
+def create_env_file(env_vars: dict[str, str], path: Path) -> None:
+    lines = [f"{k}={v}" for k, v in env_vars.items()]
+    path.write_text("\n".join(lines))
+
+
+def setup_logger(logger_name: str) -> logging.Logger:
```

Comment:
I just noticed that this is actually never used.. 🤔

Reply:
Yep, I had some problems with properly creating multiple new loggers that would maintain the original uvicorn formatting, so I gave up on it for now... I guess I should at least put there TODO so that we won't forget about it